server/helper-functions.js

```diff
@@ -52,11 +52,11 @@ module.exports.generateRoles = (usernames) => {
   // to roles
 
   // output array of users roles
-  var userRoles = {}
+  let userRoles = {}
   // get the length of the usernames.
-  var key = Object.keys(usernames).length;
+  const key = Object.keys(usernames).length;
   // shuffle the roles
-  var shuffleRoles = _.shuffle(roles(key));
+  const shuffleRoles = _.shuffle(roles(key));
```

Comment:
Should this really be a const? Strictly, it isn't being changed in the block scope and each invocation of generateRoles can create it anew. But, using `const` strongly suggests that the value will be constant.